### ISSUE TITLE
The Names of the Things

### ISSUE THREAD

**Issue by josephjclark** [OpenFn/kit#374]
When you know the name of a thing, you have control over it.

I am not happy with the names of:

* Runtime Manager Server - which is kinda weird because it's not a server, and doesn't reflect the close coupling with Lightning
* Runtime Manager - which is supposed to be a generic term for "a thing that wraps an interface around the runtime", meaning it handles compilation, options, autoinstall, input and output.

The server could be:
* Lightning Runtime Service
* Lightning Workflow Runner
* **Lightning Attempt Runner**

The runtime could be:
* **Runtime Worker**
* Runtime Service (Although it's not really a service, per se, it's an API. It needs wrapping up in some kind of service to manage its lifecycle)
* Runtime Module (generic, meaningless)
* Pooled Executor
* Runtime Thread Worker (too fixated on threads)
* Runtime Orchestrator

**Comment by josephjclark** [comment]
I think we have some answers to this!

![image](https://github.com/OpenFn/kit/assets/7052509/a31e9742-ac0c-42e7-8035-70113de64725)

The abstraction that Lightning speaks to is called the Worker.

The RTM Server is going to become the `ws-worker` or websocket worker. We may later have other classes of worker (for example, lambda or local dev).

The RTM becomes an `engine` - which is like a second generation of the old engine, or a thing that does the work.  The first one, the threadworking one will be called `engine-multi`.

**Comment by taylordowns2000** [comment]
Does this all sound OK to you @josephjclark ?

1. Check the kubernetes cluster, we've got two node pools... one is optimized for Lightning instances and the other for Worker instances.
2. How many workers are connected? ... "There are 7 online right now."
3. Tell me more... what style of worker are they? ... "We're using the default worker implementation... that's `ws-worker` and `engine-multi`"
4. How many attempts can we process concurrently then? ... "Well, based on the CPU and RAM we've allocated to each worker, we've decided to limit them to 10 threads at a time. So those 7 workers can process 70 attempts concurrently."
5. It looks like one of the workers has gone unresponsive. We're down to 6. We should set up a health check endpoint so that Kubernetes can restart that node.

Do all of these little interactions sound OK to you?
